web/src/components/Collapsible.tsx

```diff
@@ -53,10 +53,9 @@ const Collapsible = ({
   return (
     <Container className={className}>
       <CollapsibleHeader
-        role='button'
+        ariaLabel={t(collapsed ? 'showMore' : 'showLess')}
```

Comment:
Better use the aria-expanded, which is explicitly for collapiables: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

Reply:
Good suggestion. However, I think the `ariaLabel` should still not be removed. In the example both is used together. Will add the `aria-expanded` though.

Reply:
I agree